packages/vite/src/node/server/index.ts

```diff
@@ -115,6 +115,12 @@ export interface ServerOptions extends CommonServerOptions {
    * @default true
    */
   preTransformRequests?: boolean
+  /**
+   * Whether or not to ignore-list source files in the dev server sourcemap, used to populate
+   * the [`x_google_ignoreList` source map extension](https://developer.chrome.com/blog/devtools-better-angular-debugging/#the-x_google_ignorelist-source-map-extension).
+   * By default, it excludes all paths containing `/node_modules/`.
+   */
+  sourcemapIgnoreList?: (sourcePath: string, sourcemapPath: string) => boolean
```

Comment:
Per suggestion from @lukastaegert, I updated https://github.com/rollup/rollup/pull/4877 to allow for `boolean`s here as well, meaning you can disable the ignore-listing completely via `sourcemapIgnoreList: false`. I think it'd be good to also use the same logic here in vite then.